### DISCUSSION TITLE
[ FF7 ] Game crash when entering in a city from world map

### DISCUSSION THREAD

**Discussion by ramayana0** [julianxhokaxhiu/FFNx#214] [Q&A]
**Describe the bug**

After leaving Midgard and when entering Kalm the game crashes with the following message box 
"Oops something very bad happened..."

Reproduced when going to Chocobo farm.

Here's the two files asked by the message box :
[bug.zip](https://github.com/julianxhokaxhiu/FFNx/files/6096236/bug.zip)

**To Reproduce**

1. Load a save on the world map near Kalm
2. Go to Kalm
3. The game crashes with the battle music triggering
4. Message box appears

**Expected behavior**

When entering Kalm from World Map, the city should load.

**GPU (please complete the following information):**

- Nvidia GTX 1070
- Driver Version 27.21.14.5671

Thanks for your work, that's amazing.

**Comment by julianxhokaxhiu** [comment]
Please update to the latest canary, and try again.

**Comment by ramayana0** [comment]
After changing  it's ok but i can't change configuration and only go with default, and only windowed, but i suppose this is normal. Thanks.

**Comment by julianxhokaxhiu** [comment]
> After changing it's ok but i can't change configuration and only go with default, and only windowed, but i suppose this is normal. Thanks.

You can customize everything by editing the `FFNx.toml` file :) Welcome, enjoy!
